server/routes/user.js

```diff
@@ -4,17 +4,55 @@ const User = require("../models/User");
 
 // Find user
 router.get("/", async (req, res, next) => {
-  console.log(req.body)
+  console.log(req.body);
   const { id } = req.body;
   try {
     const user = await User.findUser(id);
-    res.status(201).json({ msg: user });
-    }
-  catch (err) {
+    res.status(201).send(user);
+  } catch (err) {
+    res.status(404).send({ msg: err });
+  }
+});
+
+// Find user by email
+router.post("/", async (req, res, next) => {
+  const { email } = req.body;
+  try {
+    const user = await User.findByEmail(email);
+    res.status(201).send(user);
+  } catch (err) {
     res.status(404).send({ msg: err });
   }
 });
 
+// Filter users to share with
+router.post("/filter", async (req, res, next) => {
```

Comment:
This should also be a GET route, maybe a better path for it would be something like "/suggestions"